template/src/components/custom-drawer.tsx

```diff
@@ -2,8 +2,9 @@ import { DrawerContentComponentProps } from '@react-navigation/drawer';
 import { observer } from 'mobx-react-lite';
 import React from 'react';
 import { StyleSheet, View } from 'react-native';
-import { Avatar, Drawer, Text, ToggleButton } from 'react-native-paper';
+import { Avatar, Drawer, Text, ToggleButton, useTheme } from 'react-native-paper';
 import FixedContainer from '~/components/fixed-container';
+import { IMAGESIZE, WIDTH } from '~/constant';
```

Comment:
As stated, these should be computed in the component itself with `useDimensions` hook. Also, as they are not constants, the names should be in `small-caps`.